pkg/backup/backup/backup_manager.go

```diff
@@ -110,9 +111,71 @@ func (bm *backupManager) syncBackupJob(backup *v1alpha1.Backup) error {
 		return controller.IgnoreErrorf("invalid backup spec %s/%s cause %s", ns, name, err.Error())
 	}
 
-	_, err = bm.deps.JobLister.Jobs(ns).Get(backupJobName)
```

Comment:
put them to a small function, don't make it worse

Reply:
Can you refactor this function? The code is bad before you add code into it, it become more complicate when you add more code.

Reply:
ok, i can refactor this code in next commit.